src/Security/MemberAuthenticator/ChangePasswordForm.php

```diff
@@ -50,19 +53,15 @@ public function __construct($controller, $name, $fields = null, $actions = null)
      */
     protected function getFormFields()
     {
-        $fields = FieldList::create();
-
+        $field = ConfirmedPasswordField::create('Password');
```

Comment:
Label should still be "New Password" I think

Reply:
Updated